src/main/java/com/redhat/devtools/lsp4ij/settings/jsonSchema/LSPServerConfigurationJsonSchemaFileProvider.java

```diff
@@ -0,0 +1,84 @@
+package com.redhat.devtools.lsp4ij.settings.jsonSchema;
+
+import com.intellij.openapi.application.ModalityState;
+import com.intellij.openapi.application.WriteAction;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.psi.impl.PsiManagerEx;
+import com.intellij.psi.impl.file.impl.FileManagerImpl;
+import com.intellij.util.ModalityUiUtil;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Objects;
+
+public class LSPServerConfigurationJsonSchemaFileProvider extends AbstractLSPJsonSchemaFileProvider {
+
+    private static final Object SCHEMA_CHANGE_REQUESTOR = new Object();

```

Comment:
For VFS operations that have a `requestor`, I generally provide either the instance (typically `this`) or the containing class, e.g., `LSPServerConfigurationJsonSchemaFileProvider.class`. There's no need for a distinct `Object` for that parameter.

Reply:
I don't use request anymore